web/source/osk/visualKeyboard.ts

```diff
@@ -1778,6 +1778,18 @@ namespace com.keyman.osk {
       if(this.inputEngine) {
         this.inputEngine.unregisterEventHandlers();
       }
+
+      if(this.deleting) {
+        window.clearTimeout(this.deleting);
+      }
```

Comment:
I wonder if we should do this -- not that it should make a difference after this but it doesn't then leave a dangling reference to a cleared timeout?

```suggestion
      if(this.deleting) {
        window.clearTimeout(this.deleting);
        this.deleting = 0;
      }
```

FWIW, `deleting` is not a good variable name! But that shouldn't be tackled here.